inlong-common/src/main/java/org/apache/inlong/common/pojo/sort/dataflow/field/format/DateFormatInfo.java

```diff
@@ -98,11 +99,10 @@ public String serialize(Date date) {
                 return Long.toString(seconds);
             }
             default: {
-                if (simpleDateFormat == null) {
+                if (formatter == null) {
                     throw new IllegalStateException();
                 }
-
-                return simpleDateFormat.format(date);
+                return date.toLocalDate().format(formatter);
```

Comment:
Date.toLocalDate is deprecated.
It is SortFlink job timezone, it is not business data timezone.

Reply:
Fixed, use thread local to ensure the thread safety.